create release branch later in cycle (#1085)
kubernetes/sig-release: release-team/role-handbooks/docs/README.md

---

#### Create the release branch

> Creating the release brach allows you to easily sunset the current docs under it's new branch `release-[current release]` after we merge `dev-[future release]`. For example if `master` represents `v1.20` you would create `release-1.20`.

Reviewer: ```suggestion
Creating the release branch lets you snapshot the current docs in a new branch, `release-[current release]`, after merging `dev-[future release]`. For example: if `master` represents `v1.20`, you would create `release-1.20`.
```

> > Note: this requires someone with write access to `k/website` such as a Co-Chair.

Reviewer: ```suggestion
> **Note:** Creating a branch requires someone with write access to `k/website`, such as a [SIG Docs co-chair](https://github.com/kubernetes/community/tree/master/sig-docs#leadership).
```

- From [k/website](https://github.com/kubernetes/website) click on `master` branch.
- Type the name of the release branch.
- **click** Create branch release-x.xx from master

![](pics/new-branch.png)

> Note: if the release branch is created before the website is frozen you may need to merge in master to keep up-to-date

To merge `master` into `release-[current release]` on your local fork:

```
$ git remote add upstream https://github.com/kubernetes/website.git
$ git fetch upstream
$ git checkout --track upstream/release-[current release]
$ git merge upstream/master
$ git checkout -b merged-master-release-[current release]
$ git commit -m "merged master into release-[current release] to keep in sync"
$ git push origin merged-master-release-[current release]
```

#### Update Netlify

> Update the Netlify (contact a [SIG Docs chair](https://github.com/kubernetes/community/tree/master/sig-docs#leadership) if you do not have access and they can assist with this):

Reviewer: ```suggestion
Update the Netlify configuration. (A [SIG Docs chair](https://github.com/kubernetes/community/tree/master/sig-docs#leadership) can assist you with access):
```
